pkg/namesgenerator/names-generator.go

```diff
@@ -1,3 +1,12 @@
+/*
+
+This package is officially "frozen" - no new additions will be accepted.
+
+For a long time, this package provided a lot of joy within the project, but at some point the conflicts of opinion became greater than the added joy.
+
+At some future time, this may be replaced with something that sparks less controversy, but for now it will remain as-is.
+
+*/
```

Comment:
My IDE keeps complaining that the comment (being a package-level comment) should start with `Package namesgenerator ...` (I'm sure some linters could complain as well).

While at it, perhaps change to `//` comments (looks like we're using that just about everywhere), and wrap it to ~80 chars


```suggestion
// Package namesgenerator generates random names.
// 
// This package is officially "frozen" - no new additions will be accepted.
// 
// For a long time, this package provided a lot of joy within the project, but
// at some point the conflicts of opinion became greater than the added joy.
// 
// At some future time, this may be replaced with something that sparks less
// controversy, but for now it will remain as-is.
```